utils/ansible/ftest/library/daos_hugepages.py

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+__copyright__ = """
+(C) Copyright 2018-2022 Intel Corporation.
+
+SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+
+
+ANSIBLE_METADATA = {
+    'metadata_version': '0.1',
+    'status': ['preview'],
+    'supported_by': 'Intel'
+}
+
+
+DOCUMENTATION = '''
+---
+module: daos_hugepages
+
+short_description: Configure Linux Huge Pages
+
+version_added: "0.1"
+
+description:
+    - Check if the Linux Huge Pages feature is enabled
+    - Check the value of the Huge Pages
+    - Set the value of the Huge Pages if needed
+
+options:
+    size:
+        description:
+            - Number of huge pages to allocate
+        type: int
+        default: 4096
+
+author:
+    - DAOS group <daos@daos.groups.io>
+'''
+
+EXAMPLES = '''
+- name: Set the Linux Huge Pages size
+  daos_hugepages
+'''
+
+RETURN = '''
+elapsed:
+     description: the number of seconds that elapsed while checking and setting the huge pages size
+     returned: always
+     type: int
+     sample: 3
+'''
+
+import os
+import re
+import datetime
+import subprocess
+
+from ansible.module_utils.basic import AnsibleModule
```

Comment:
(pylint-no-name-in-module) No name 'module_utils' in module 'ansible'
(pylint-import-error) Unable to import 'ansible.module_utils.basic'